common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -435,6 +468,19 @@ def definition_to_xml(self, resource_fs):
             'download_track': json.dumps(self.download_track),
             'download_video': json.dumps(self.download_video),
         }
+
+        if hasattr(settings, 'FEATURES') and settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False):
+            if self.license:
+                attrs['license'] = self.license.kind
+                attrs['license_version'] = self.license.version
+            elif hasattr(self.descriptor.runtime, 'modulestore'):
```

Comment:
`    476: E1101: (no-member), VideoDescriptor.definition_to_xml: Instance of 'VideoDescriptor' has no 'descriptor' member
`